.gitignore

```diff
@@ -96,3 +96,6 @@ venv/*
 
 # apache release artifacts
 dev/dist
+
+# logs
+logs/
```

Comment:
Running Ballista in any way seems to put logs here that aren't gitignored, which is a pain for developers to not accidentally get them into PRs.